### PR TITLE
Function to count constrants and vowels fixes issue #29  and issue #31

### PR DESCRIPTION
**Pull request by lucynao**
Function to count constrant andvowels fix issue fixes#29

### CODE REVIEW

**Comment by Veroniush** [comment]
Very nice code.
THANKS!

**Review by Veroniush — COMMENTED** [review]
Everything fine it;s okay

File: StringHelper.cs [hunk 1]
```diff
@@ -60,46 +60,57 @@ public static int CountSentenses(string text)
...
+                if (char.IsLetter(chars[i]) == true &&( chars[i].ToString().ToLower() == "a" || chars[i].ToString().ToLower() == "e" || chars[i].ToString().ToLower() == "i" || chars[i].ToString().ToLower() == "o" || chars[i].ToString().ToLower() == "u" || chars[i].ToString().ToLower() == "y"))
+                {
+                    numberOfVowes = numberOfVowes + 1;
+                }
+            }
+            return numberOfVowes;
+        }
+
+        public static int CountConstatnts(string text)
+
```
Comment (lucynao) [inline StringHelper.cs]: it is good practice not to have new line between function name and brackets. please remove if you can

File: StringHelper.cs [hunk 2]
```diff
@@ -60,46 +60,57 @@ public static int CountSentenses(string text)
...
+                if (char.IsLetter(chars[i]) == true)
                 {
                     numberOfLetters = numberOfLetters + 1;
                 }
             }
-            return numberOfLetters;
+            return numberOfLetters - StringHelper.CountVowels(text);
         }
-
         /// <summary>
```
Comment (lucynao) [inline StringHelper.cs:93]: please remove this comment

**Review by lucynao — COMMENTED** [review]
just refactoring comments

File: StringHelper.cs (hunk 2 above)
Comment (Veroniush) [reply StringHelper.cs:93]: fixed :>

**Comment by lucynao** [comment]
fixes issue #31

**Pull request merged**